lib/runc_sandbox.ml

```diff
@@ -106,7 +103,8 @@ module Json_config = struct
     in
     `Assoc fields
 
-  let make {Config.cwd; argv; hostname; user; env; mounts; network; mount_secrets} t ~config_dir ~results_dir : Yojson.Safe.t =
+  let make {Config.cwd; argv; hostname; user; env; mounts; network; mount_secrets; entrypoint} t ~config_dir ~results_dir : Yojson.Safe.t =
+    assert (entrypoint = None);
```

Comment:
I would prefer to avoid this assertion using a better type, but if that is not possible can this provide some context as to how this might occur and what to look for if it happens.

Reply:
The entrypoint is never used for the runc sandbox backend, however with the Docker backend I've found it useful to have a dedicated field to set it, to change the shell or call programs without going through the shell.